README.md

```diff
@@ -113,6 +113,30 @@ We have standardized on a few abbreviations that are allowed to use:
 - `ui` for user interface
 - `util` or `utils` for utility or utilities, as in `StringUtils`
 
+```js
+// bad
+onPrevClick();
+
+// good
+onPreviousClick();
```

Comment:
I'm not sure how we usually feel about this but onPreviousClick also doesn't mean anything and seems more like an event than a method. What does it do?

Reply:
> and seems more like an event than a method

That's exactly what it is (and event handler), so since you picked that up, I would say it's named appropriately :)

Reply:
Wouldn't you prefer to know what it does against what it is? Or is it something you'd prefer to have in the doc above the method?